tests/components/unifi/test_sensor.py

```diff
@@ -134,43 +164,82 @@ async def test_sensors(hass):
     wireless_client_tx = hass.states.get("sensor.wireless_client_name_tx")
     assert wireless_client_tx.state == "6789.0"
 
+    wireless_client_uptime = hass.states.get("sensor.wireless_client_name_uptime")
+    assert wireless_client_uptime.state == "2020-09-15T14:41:00+00:00"
+
+    wired_client_uptime = hass.states.get("sensor.wired_client_name_uptime")
+    assert wired_client_uptime.state == "2020-09-14T14:41:45+00:00"
+
+    # Try to add the sensors again, using a signal
+    clients_connected = set()
+    devices_connected = set()
+
+    clients_connected.add(clients[0]["mac"])
+    clients_connected.add(clients[1]["mac"])
+
+    async_dispatcher_send(
+        hass,
+        controller.signal_update,
+        clients_connected,
+        devices_connected,
+    )
+
+    await hass.async_block_till_done()
+
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 6
+
 
 async def test_remove_sensors(hass):
     """Test the remove_items function with some clients."""
     controller = await setup_unifi_integration(
         hass,
-        options={CONF_ALLOW_BANDWIDTH_SENSORS: True},
+        options={
+            CONF_ALLOW_BANDWIDTH_SENSORS: True,
+            CONF_ALLOW_UPTIME_SENSORS: True,
+        },
         clients_response=CLIENTS,
     )
-    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 4
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 6
     assert len(hass.states.async_entity_ids(TRACKER_DOMAIN)) == 2
 
     wired_client_rx = hass.states.get("sensor.wired_client_name_rx")
     assert wired_client_rx is not None
     wired_client_tx = hass.states.get("sensor.wired_client_name_tx")
     assert wired_client_tx is not None
 
+    wired_client_uptime = hass.states.get("sensor.wired_client_name_uptime")
+    assert wired_client_uptime is not None
+
     wireless_client_rx = hass.states.get("sensor.wireless_client_name_rx")
     assert wireless_client_rx is not None
     wireless_client_tx = hass.states.get("sensor.wireless_client_name_tx")
     assert wireless_client_tx is not None
 
+    wireless_client_uptime = hass.states.get("sensor.wireless_client_name_uptime")
+    assert wireless_client_uptime is not None
+
     controller.api.websocket._data = {
         "meta": {"message": MESSAGE_CLIENT_REMOVED},
         "data": [CLIENTS[0]],
     }
     controller.api.session_handler(SIGNAL_DATA)
     await hass.async_block_till_done()
 
-    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 2
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)